pay-queue/wsgi.py

```diff
@@ -11,9 +14,13 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Enum definitions."""
+"""Provides the WSGI entry point for running the application."""
+import os
+from pay_queue import create_app
+
+
+app = create_app()
 
-CAS_MESSAGE_TYPE = 'bc.registry.payment.casSettlementUploaded'
-CGI_ACK_MESSAGE_TYPE = 'bc.registry.payment.cgi.ACKReceived'
-CGI_FEEDBACK_MESSAGE_TYPE = 'bc.registry.payment.cgi.FEEDBACKReceived'
-EFT_MESSAGE_TYPE = 'bc.registry.payment.eft.fileUploaded'
+if __name__ == '__main__':
+    server_port = os.environ.get('PORT', '8080')
```

Comment:
Hmm 8080, would recommend 5001 or 5002? I think pay-api is 5000.

Reply:
fixed